tests/suites/casper/popup.js

```diff
@@ -75,17 +96,23 @@ casper.test.begin('popup tests', 22, function(test) {
         this.removeAllListeners('popup.closed');
     });
 
-    casper.thenClick('a[target="_blank"]');
+    casper.thenClick('a[target="form"]');
 
     casper.waitForPopup('form.html', function() {
-        test.pass('Casper.waitForPopup() waits when clicked on a link with target=_blank');
+        test.pass('Casper.waitForPopup() waits when clicked on a link with target=form');
     });
 
     casper.withPopup('form.html', function() {
         test.assertTitle('CasperJS test form');
     });
 
+    casper.withPopup(0, function() {
+        test.assertTitle('CasperJS test form');
+        //this.page.close();
```

Comment:
I had let this line because CasperJS doesn't close webPage correctly.
I will propose a PR to fix this bug but you are right I have to clean this line for now.